app/constants.py

```diff
@@ -3,15 +3,15 @@
 
 from .ntypes import NEWRL_TOKEN_CODE, NUSD_TOKEN_CODE
 
-SOFTWARE_VERSION = "1.3.0"
+SOFTWARE_VERSION = "1.3.1"
 
 NEWRL_ENV = os.environ.get('NEWRL_ENV')
 IS_TEST = os.environ.get('NEWRL_TEST') == 'Y'
 
 if NEWRL_ENV == 'testnet':
-    BOOTSTRAP_NODES = ['bootstrap1-testnet1.newrl.net']
-    NETWORK_TRUSTED_ARCHIVE_NODES = ['archive1-testnet1.newrl.net']
-    NEWRL_PORT = 8421
+    BOOTSTRAP_NODES = ['bootstrap1-lakeshore.newrl.net']
```

Comment:
@Prannav-k let's use a new NEWRL_ENV as `lakeshore` for this. else current testnet will have issue when this is merged.